### ISSUE TITLE
Single-line comment parsing issue

### ISSUE THREAD

**Issue by Wuerfel21** [ironsheep/P2-vscode-extensions#5]
**Checklist:**

- [x ] I updated to the latest spin2 version available
- [ x] I have restarted VSCode and the problem still exists

**Version of Spin2 Extension with the issue:** the latest one

**Last working version (if known):** I think it did work at some point, but I know the previous version was busted already

**Hardware, Operating System, VSCode version:** Win7 on amd64, presumably latest VSC

**Description of problem:**

Single-line comments that follow code aren't recognized as comments anymore. Happens in every file, but here's an example in megayume (latest commit on beta branch)
![image](https://user-images.githubusercontent.com/6944842/164993528-aeece863-117d-4a77-b7b4-596f06cd9813.png)

**Comment by ironsheep** [comment]
OK, not exactly the problem as described but a ramification of some of the comments being highlighted during semantic processing instead of during syntax processing.  When Chip decided to use the comment character (') (single-quote) to additionally mark strings in debug() statements we can no longer treat the single quotes without knowing the semantic meaning. 

So while the comment is marked as a comment correctly during semantic processing and even highlights correctly for our own theme "**Spin2 IronSheep**":

![Screen Shot 2022-04-25 at 00 11 56](https://user-images.githubusercontent.com/540005/165031026-fff1e3da-d02c-4088-93cd-c2840320e42d.png)

 ...the comment marking appears to not be fully described enough to be seen correctly by other themes?  Here's more detail on what I mean:

First, comment found during syntax processing:
![syntax-comment](https://user-images.githubusercontent.com/540005/165032193-f0135f16-02f4-447d-ae14-b9661437e11a.png)

Second, comment found during semantic processing:
![semantic-comment-2](https://user-images.githubusercontent.com/540005/165032299-6f2a26d5-12d9-423a-b517-52756cdde8cf.png)

So far I've been unable to find a way to influence the "Standard Token Type" value during semantic processing.  I think this is why the theme you chose does not see the comment correctly.  But I'll continue to look.

@Wuerfel21   Please tell me which theme this is and I'll see if I can make it register correctly...

**Comment by Wuerfel21** [comment]
The theme I'm using is this one: https://marketplace.visualstudio.com/items?itemName=k3a.theme-dark-plus-contrast
... which is just the default VSC theme but with black background such as to not be an assault on the eyeballs.

**Comment by Wuerfel21** [comment]
Revelations:
- The actual default theme doesn't have the issue
- The theme I like actually does more than just changing the BG color

So I guess I'll have to recreate the dark+ contrast theme from scratch. Oh well...

**Comment by Wuerfel21** [comment]
Doing color overrides over the default theme in user config seems to do the trick:
```json
    "workbench.colorCustomizations": {
        "[Default Dark+]": {
            "editor.background": "#000000",
            "editor.foreground": "#ffffff",
            "sideBarSectionHeader.background": "#383838",
            "sideBarSectionHeader.border": "#383838",
            "editorIndentGuide.background": "#50505080",
            "editorIndentGuide.activeBackground": "#80808080",
        },
    },
    "editor.tokenColorCustomizations": {
        "[Default Dark+]": {
            "strings": "#F3B499",
            "textMateRules": [
                {
                    "scope": "constant.language",
                    "settings": {
                        "foreground": "#74C0FF"
                    }
                }
            ]
        }
    },
```
![image](https://user-images.githubusercontent.com/6944842/165268907-7f16e255-cfa9-4ac4-95b4-cd5639641e1a.png)

**Comment by ironsheep** [comment]
Fixed, Changes appear in v1.4.1 and later.
